ecms_base/ecms_base.install

```diff
@@ -2132,3 +2132,16 @@ function ecms_base_update_10200(array &$sandbox): void {
     $active_storage->write("{$config}", $install_source->read("{$config}"));
   }
 }
+
+/**
+ * Add ecms_ckeditor to the installed modules.
+ */
+function ecms_base_update_10201(array &$sandbox): void {
```

Comment:
Just a note that 10201 might have already been used in another update for this release. Might have a merge conflict at some point.